website/docusaurus.config.js

```diff
@@ -1,5 +1,7 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
+require('dotenv').config();
```

Comment:
GitHub was lagging earlier so you probably didn't see my last commit, but we don't actually need this or the dep on dotenv.

Reply:
Sounds good. I'll nix it!